api/v1alpha1/instancegroup_types.go

```diff
@@ -392,6 +392,13 @@ func (s *EKSSpec) Validate() error {
 		s.Type = LaunchConfiguration
 	}
 
+	if s.Type == LaunchConfiguration {
+		if s.EKSConfiguration.MixedInstancesPolicy != nil {
+			log.Info("cannot use mixedInstancesPolicy with LaunchConfiguration, will ignore provided mixedInstancePolicy")
```

Comment:
Not sure if this log is going to be very useful - does it contain any identifying information for the IG?

Might also make sense to event this on the IG, for users that aren't cluster operators.

Reply:
Yeah, we should pass the IG name from the top level `Validate()` so we can add the IG name.
Regarding events - yeah, that's a great idea to have events for any IG error associated with the IG for visibility - we might want to do that as a separate effort though across the board.

Reply:
Created https://github.com/keikoproj/instance-manager/issues/304 to address events